.github/eval.sh

```diff
@@ -0,0 +1,182 @@
+    echo " -t    Filter evaluation targets (default='packages')"
+    echo " -v    Set the script verbosity to DEBUG"
+    echo " -h    Print this help message"
+    echo ""
+    echo "Examples:"
+    echo ""
+    echo "  Following four commands (combined) will evaluate flake outputs that "
+    echo "  match the default filter 'packages' - each command should be executed"
+    echo "  in its own github runner, thus splitting the eval work on "
+    echo "  separate hosts allowing concurrent execution:"
+    echo ""
+    echo "    $MYNAME -j 0 -m 4"
+    echo "    $MYNAME -j 1 -m 4"
+    echo "    $MYNAME -j 2 -m 4"
+    echo "    $MYNAME -j 3 -m 4"
+    echo ""
+}
+
+################################################################################
+
+on_exit () {
+    rm -fr "$TMPDIR" # remove tmpdir
+}
+
+on_err () {
+    kill 0 # kill the master shell and possible subshells
+}
+
+print_err () {
+    printf "${RED}Error:${NONE} %b\n" "$1" >&2
+}
+
+argparse () {
+    # Colorize output if stdout is to a terminal (and not to pipe or file)
+    if [ -t 1 ]; then RED='\033[1;31m'; NONE='\033[0m'; fi
+    # Parse arguments
+    JOB_ID=""; MAX_JOBS=""; EVAL_TARGETS="packages"; OPTIND=1
+    while getopts "hvj:m:t:" copt; do
+        case "${copt}" in
+            h)
+                usage; exit 0 ;;
+            v)
+                set -x ;;
+            t)
+                EVAL_TARGETS="$OPTARG" ;;
+            j)
+                JOB_ID="$OPTARG"
+                if ! [[ "$JOB_ID" == +([0-9]) ]]; then
+                    print_err "'-j' expects a non-negative integer (got: '$JOB_ID')"
+                    usage
+                fi
+                ;;
+            m)
+                MAX_JOBS="$OPTARG"
+                if ! [[ "$MAX_JOBS" == +([0-9]) ]]; then
+                    print_err "'-m' expects a non-negative integer (got: '$MAX_JOBS')"
+                    usage
+                fi
+                ;;
+            *)
+                print_err "unrecognized option"; usage; exit 1 ;;
+        esac
+    done
+    shift $((OPTIND-1))
+    if [ -n "$*" ]; then
+        print_err "unsupported positional argument(s): '$*'"; exit 1
+    fi
+    if [ -z "$JOB_ID" ]; then
+        print_err "missing mandatory option (-j)"; usage; exit 1
+    fi
+    if [ -z "$MAX_JOBS" ]; then
+        print_err "missing mandatory option (-m)"; usage; exit 1
+    fi
+    if [ "$JOB_ID" -ge "$MAX_JOBS" ]; then
+        print_err "'-j' must be smaller than '-m'"; usage; exit 1
+    fi
+}
+
+exit_unless_command_exists () {
+    if ! command -v "$1" &>/dev/null; then
+        print_err "command '$1' is not installed (Hint: are you inside a nix-shell?)"
+        exit 1
+    fi
+}
+
+################################################################################
+
+evaluate () {
+    job="$1"
+    max_jobs="$2"
+    filter="$3"
+    echo "[+] Using filter: '$filter'"
+    # Output all flake output names
+    nix flake show --all-systems --json |\
```

Comment:
If this is just about evaluation, this will already evaluate everything. You shouldn't need nix-eval-jobs afterwards.

Reply:
It will be also faster because nix-eval-jobs has to serialize drv files to disk.